apps/glusterfs/listing.go

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"github.com/boltdb/bolt"
+
+	"github.com/heketi/heketi/pkg/glusterfs/api"
+)
+
+// ListCompleteVolumes returns a list of volume ID strings for volumes
+// that are not pending.
+func ListCompleteVolumes(tx *bolt.Tx) ([]string, error) {
```

Comment:
Just a suggestion, how about ListCompletedVolumes ?

Reply:
That's fine too. I'll let @obnoxxx chime in with which version is preferred as a tiebreaker.

Reply:
imho , let's stick with complete (1 char shorter...) ;-)